.golangci.yml

```diff
@@ -1,27 +1,59 @@
 # Copyright KubeArchive Authors
 # SPDX-License-Identifier: Apache-2.0
 ---
+version: '2'
 run:
-  timeout: 10m
   build-tags:
     - integration
 linters:
   enable:
     - bodyclose
     - containedctx
     - fatcontext
-    - goimports
     - gosec
     - nilerr
     - nilnil
+    - unparam
     - usestdlibvars
     - wastedassign
-    - unparam
-linters-settings:
-  govet:
-    enable-all: false
-    disable-all: false
-    enable:
-      - loopclosure
-      - nilness
-      - shadow
+  settings:
+    staticcheck:
+      checks:
+        # defaut
+        - all
+        - -ST1000
+        - -ST1003
+        - -ST1016
+        - -ST1020
+        - -ST1021
+        - -ST1022
+        # /default
+        - -QF1003 # use switch instead of if/ifelse/else
+        - -ST1005 # do not capitalize errors
```

Comment:
I'd keep this one, we already had some issues at some point trying to refactor the errors that we had that weren't following this guideline: https://github.com/kubearchive/kubearchive/issues/536

Reply:
We did not enforce it for everything, just for `abort.Abort`. I'm OK with that, but I don't remember if we got to an agreement of some sorts. I enabled it and fixed all the errors in a separate commit.